src/components/vive-controls.js

```diff
@@ -6,18 +6,20 @@ var VIVE_CONTROLLER_MODEL_OBJ_MTL = 'https://cdn.aframe.io/controllers/vive/vr_c
 
 /**
  * Vive Controls Component
- * Interfaces with vive controls and maps Gamepad events to
- * vive controller buttons: trackpad, trigger, grip, menu and system
- * It loads a vive controller model and highlights the pressed buttons
+ * Interfaces with vive controllers and maps Gamepad events to
+ * common controller buttons: trackpad, trigger, grip, menu and system
+ * It loads a controller model and highlights the pressed buttons
  */
 module.exports.Component = registerComponent('vive-controls', {
   dependencies: ['tracked-controls'],
 
   schema: {
+    idPrefix: { default: 'OpenVR Gamepad' },
```

Comment:
This should not be part of the schema since it won't be configurable. It should be a private const within the component like https://github.com/aframevr/aframe/pull/2132/files#diff-3056bccc97756be52ace666f5e213434R6